rest-api/calamari_rest/urls/v2.py

```diff
@@ -75,6 +75,11 @@
                                                     'patch': 'update',
                                                     'delete': 'destroy'}),
         name='cluster-pool-detail'),
+    url(r'^cluster/(?P<fsid>[a-zA-Z0-9-]+)/pool/(?P<pool_id>\d+)/stats$',
```

Comment:
@b-ranto please add a list view at /api/cluster/FSID/pool/stats you're almost there